tests/e2e/test_api.py

```diff
@@ -76,6 +81,14 @@ def setup_module(module):
 
         client = client_utils.get_http_client(ols_url, token)
         metrics_client = client_utils.get_http_client(ols_url, metrics_token)
+
+        # Wait for OLS to be ready
+        success = wait_for_ols(ols_url)
+        if not success:
+            generate_junit_report(os.getenv("SUITE_ID"), False)
```

Comment:
does the pytest.fail below not already guarantee a junit test entry being produced?  (put another way, what does the junit + test job output look like if you don't explicitly do this?)

Reply:
oh okay, Will just remove it then